code/game/gamemodes/gang/gang.dm

```diff
@@ -69,20 +94,55 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 	if(gangs.len < 2) //Need at least two gangs
 		return 0
 
-	return 1
+	//turn off sec and captain
+	SSjob.DisableJob(/datum/job/captain)
+	SSjob.DisableJob(/datum/job/hos)
+	SSjob.DisableJob(/datum/job/warden)
+	SSjob.DisableJob(/datum/job/detective)
+	SSjob.DisableJob(/datum/job/officer)
 
+	return 1
 
-/datum/game_mode/gang/post_setup()
+/datum/game_mode/gang/send_intercept()
 	set waitfor = FALSE
-	..()
-	sleep(rand(10,100))
+	..(FALSE, FALSE)
+
+	priority_announce("Incoming Bluespace Artillery Strike", "Target location: Security department. Evacuate immediately", 'sound/machines/Alarm.ogg')
+
+	sleep(100)
+
+	explosion(target_armory, 7, 14, 28, 30, TRUE, TRUE)
+	explosion(target_equipment, 7, 14, 28, 30, TRUE, TRUE)
+
+	sleep(1200)
+
+	var/list/bosses = list()
+
 	for(var/datum/gang/G in gangs)
-		for(var/datum/mind/boss_mind in G.bosses)
-			G.add_gang_hud(boss_mind)
-			forge_gang_objectives(boss_mind)
-			greet_gang(boss_mind)
-			equip_gang(boss_mind.current,G)
-			modePlayer += boss_mind
+		for(var/boss_mind in G.bosses)
+			bosses += boss_mind
+	for(var/I in bosses)
+		var/datum/mind/boss_mind = I
+		var/list/other_bosses = bosses.Copy() - boss_mind
+		var/msg = "<span class='danger'>You've decided to take charge of the situation and form a gang."
+		for(var/J in 1 to other_bosses.len)
+			var/datum/mind/other_mind = other_bosses[J]
+			if(J == other_bosses.len)
+				msg += " and"
+			msg += " J"
+			if(J != other_bosses.len)
+				msg += ","
+		to_chat(boss_mind.current, "[msg] are your ganghead rivals. [syndicate_name()] has offered an uplink to help you take over, it will be delivered in five minutes.</span>")
```

Comment:
the syndicate name should include the gang's name, so you get, like, cybersun supporting the cyber gang, etc